### PR TITLE
New blog

### PR DESCRIPTION
**Pull request by xenova**

### CODE REVIEW

File: gemma-july-update.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
+---
+title: "PUT YOUR TITLE HERE" 
+thumbnail: /blog/assets/gemma-july-update/thumbnail.png
+authors:
+- user: Xenova
+- user: reach-vb
```
Comment (Vaibhavs10) [inline gemma-july-update.md:6]:
```suggestion
- user: pcuenq
- user: reach-vb
```

File: gemma-july-update.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+---
+
+# Google releases Gemma 2 2B, ShieldGemma and Gemma Scope.
+
+One month after the release of [Gemma 2](https://huggingface.co/blog/gemma2), Google has expanded their set of Gemma models to include the following new additions:
+- [Gemma 2 2B](https://huggingface.co/collections/google/gemma-2-2b-release-66a20f3796a2ff2a7c76f98f) - The 2.6B parameter version of Gemma 2, making it a great candidate for on-device use.
+- [ShieldGemma](https://huggingface.co/collections/google/shieldgemma-release-66a20efe3c10ef2bd5808c79) - A series of safety classifiers, trained on top of Gemma 2, for developers to filter inputs and outputs of their applications.
+- [Gemma Scope](https://huggingface.co/collections/google/gemma-scope-release-66a4271f6f0b4d4a9d5e04e2) - A comprehensive, open suite of sparse autoencoders for Gemma 2 2B and 9B.
```
Comment (pcuenca) [inline gemma-july-update.md:14]: I know this comes from the landing page, but there are scopes for the 27b as well, aren't there?

File: gemma-july-update.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+}'
+```
+
+Note: The above example runs the inference using the official GGUF weights provided by Google in `fp32`. You can create and share custom quants using the [GGUF-my-repo](https://huggingface.co/spaces/ggml-org/gguf-my-repo) space!
+
+### Demo
+
+You can chat with the Gemma 2 2B Instruct model on Hugging Face Spaces! Check it out here: https://huggingface.co/spaces/huggingface-projects/gemma-2-2b-it.
+
+In addition to this you can run the Gemma 2 2B Instruct model directly from a colab here: [TODO add the link shortly before the release]
```
Comment (Vaibhavs10) [inline gemma-july-update.md:113]:
```suggestion
In addition to this you can run the Gemma 2 2B Instruct model directly from a [colab here](https://github.com/Vaibhavs10/gpu-poor-llm-notebooks/blob/main/Gemma_2_2B_colab.ipynb)
```

File: gemma-july-update.md (hunk 2 above)
Comment (xenova) [reply gemma-july-update.md:14]: I asked about this - not "comprehensive" so they didn't want to include it linked

Comment (pcuenca) [reply gemma-july-update.md:15]: ok, thanks, but I find it confusing, I'm sure others will :)

File: gemma-july-update.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+Gemma 2 2B shares the same architecture as the other models in the Gemma 2 family, and therefore leverages technical features like sliding attention and logit soft-capping. You can check more details in [this section of our previous blog post](https://huggingface.co/blog/gemma2#technical-advances-in-gemma-2). Like in the other Gemma 2 models, we recommend you use `bfloat16` for inference.
+
+### Use with Transformers
+
+With Transformers, you can use Gemma and leverage all the tools within the Hugging Face ecosystem. To use Gemma models with transformers, make sure to use `transformers` from main for the latest fixes and optimizations:
```
Comment (pcuenca) [inline gemma-july-update.md:27]:
```suggestion
With Transformers, you can use Gemma and leverage all the tools within the Hugging Face ecosystem. To use Gemma models with transformers, make sure to use `transformers` from `main` for the latest fixes and optimizations:
```

File: gemma-july-update.md [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+You can run Gemma 2 on-device (on your Mac, Windows, Linux and more) using llama.cpp in just a few minutes.
+
+Step 1: Install llama.cpp
+
+On a Mac you can directly install llama.cpp with brew. To set up llama.cpp on other devices, please take a look here: https://github.com/ggerganov/llama.cpp?tab=readme-ov-file#usage
+
+```bash
+brew install llama.cpp
+```
+Note: if you are building llama.cpp from scratch then remember to pass `LLAMA_CURL=1` flag.
```
Comment (pcuenca) [inline gemma-july-update.md:70]:
```suggestion
Note: if you are building llama.cpp from scratch then remember to pass the `LLAMA_CURL=1` flag.
```

File: gemma-july-update.md [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+}
+]
+}'
+```
+
+Note: The above example runs the inference using the official GGUF weights provided by Google in `fp32`. You can create and share custom quants using the [GGUF-my-repo](https://huggingface.co/spaces/ggml-org/gguf-my-repo) space!
+
+### Demo
+
+You can chat with the Gemma 2 2B Instruct model on Hugging Face Spaces! Check it out here: https://huggingface.co/spaces/huggingface-projects/gemma-2-2b-it.
```
Comment (pcuenca) [inline gemma-july-update.md:112]:
```suggestion
You can chat with the Gemma 2 2B Instruct model on Hugging Face Spaces! [Check it out here](https://huggingface.co/spaces/huggingface-projects/gemma-2-2b-it).
```

File: gemma-july-update.md [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+<start_of_turn>user
+LaMDA<end_of_turn>
+<start_of_turn>model
+LaMDA who?<end_of_turn><eos>
+```
+
+This format has to be exactly reproduced for effective use. We’ll later show how easy it is to reproduce the instruct prompt with the chat template available in `transformers`. 
+
+[TODO] add mention of chat template here?
+
```
Comment (pcuenca) [inline gemma-july-update.md:134]:
```suggestion
```

File: gemma-july-update.md [hunk 8]
```diff
@@ -0,0 +1,323 @@
...
+knock knock<end_of_turn>
+<start_of_turn>model
+who is there<end_of_turn>
+<start_of_turn>user
+LaMDA<end_of_turn>
+<start_of_turn>model
+LaMDA who?<end_of_turn><eos>
+```
+
+This format has to be exactly reproduced for effective use. We’ll later show how easy it is to reproduce the instruct prompt with the chat template available in `transformers`.
```
Comment (pcuenca) [inline gemma-july-update.md:131]:
```suggestion
This format has to be exactly reproduced for effective use. In [a previous section](#use-with-transformers) we showed how easy it is to reproduce the instruct prompt with the chat template available in `transformers`. 
```

**Review by pcuenca — APPROVED** [review]

File: _blog.yml [hunk 9]
```diff
@@ -4474,4 +4474,16 @@
...
     - diffusers
     - guide
-    - diffusion-transformers
\ No newline at end of file
+    - diffusion-transformers
+
+- local: gemma-july-update
+  title: "Google releases Gemma 2 2B, ShieldGemma and Gemma Scope"
+  author: Xenova
+  thumbnail: /blog/assets/gemma-july-update/thumbnail.png
```
Comment (pcuenca) [inline _blog.yml:4482]:
```suggestion
  thumbnail: /blog/assets/gemma-july-update/thumbnail.jpg
```

File: gemma-july-update.md [hunk 10]
```diff
@@ -0,0 +1,322 @@
+---
+title: "Google releases Gemma 2 2B, ShieldGemma and Gemma Scope"
+thumbnail: /blog/assets/gemma-july-update/thumbnail.png
```
Comment (pcuenca) [inline gemma-july-update.md:3]:
```suggestion
thumbnail: /blog/assets/gemma-july-update/thumbnail.jpg
```

**Pull request merged**